frontends/concrete-python/concrete/fhe/compilation/circuit.py

```diff
@@ -292,6 +298,9 @@ def decrypt(
                 decrypted result(s) of evaluation
         """
 
+        if self.configuration.simulate_encrypt_run_decrypt:
```

Comment:
no need to explain? like "# In simulate_encrypt_run_decrypt:, we skip encryption and decryption, and run will be normal python execution